rtl/__init__.py

```diff
@@ -400,26 +400,25 @@ def interactive_controlfile(self):
         dofilepath = '%s/dofile.do' % dofiledir
         obsoletepath = '%s/Simulations/rtlsim/dofile.do' % self.entitypath
         newdofilepath = '%s/dofile.do' % self.simpath
-        if not os.path.exists(dofiledir):
-            self.print_log(type='I',msg='Creating %s' % dofiledir)
-            os.makedirs(dofiledir)
-        # Property interactive_control_contents already given and new temporary
-        # file not yet created -> create new file and use that
-        if self.interactive_control_contents != '' and not os.path.isfile(newdofilepath):
-            # Check if a custom file path was given
-            if hasattr(self, '_interactive_controlfile'):
-                dofilepath = self._interactive_controlfile
-            # Give a warning if default/custom path contains a do-file already
-            if os.path.isfile(dofilepath):
-                self.print_log(type='W',msg='Interactive control file %s ignored and interactive_control_contents used instead.' % dofilepath)
-            # Write interactive_control_contents to a temporary file
-            self.print_log(type='I',msg='Writing interactive_control_contents to file %s' % newdofilepath)
-            with open(newdofilepath,'w') as dofile:
-                dofile.write(self.interactive_control_contents)
-            self._interactive_controlfile = newdofilepath
-        # No contents or path given -> use default path (or obsolete path)
-        elif not hasattr(self, '_interactive_controlfile'):
-            # Nag about obsolete stuff
+        if not hasattr(self, '_interactive_controlfile'):
```

Comment:
Added this line first, otherwise any change to `interactive_controlfile` set by the user would be overwritten by the flow